nyc/bills.py

```diff
@@ -343,10 +356,12 @@ def scrape(self, window=3, matter_ids=None):
               'Local Laws 2015': 'bill',
               'Local Laws 2017': 'bill',
               'Local Laws 2018': 'bill',
+              'Local Laws 2019': 'bill',
```

Comment:
N.b., I added two new BILL_TYPES, since these raised errors when I ran the scraper locally without a window arg.